src/app/views/datasets/DatasetsController.jsx

```diff
@@ -113,58 +108,74 @@ class DatasetsController extends Component {
+                        var value = {
+                            title: dataset.title,
+                            date: dataset.next_release,
+                            datasetURL: "/florence/datasets/"+id+"/metadata",
+                            dataset_id: id,
+                            instances: [],
+                        };
+
+                        addedDatasets.push(datasetID);
+                        values.push(value);
+                        return;
+                    }
+                });
             }
-        });
-    }
 
-    goToDatasetMetadata(props) {
-        this.props.dispatch(push(`${location.pathname}/metadata/${props.id}`));
-    }
+            values.map((value, index) => {
+                if (value.dataset_id === datasetID) {
+                    var d = new Date(instance.last_updated);
+
+                    var inst = {
+                        edition: instance.edition,
+                        date: d.toUTCString(),
+                        version: "-",
+                        url: "/florence/datasets/"+datasetID+"/instances/"+instance.id+"/metadata",
+                    };
+
+                    value.instances.push(inst);
+
+                    values.splice(index, 1);
+                    values.push(value);
+                }
+            });
+        });
 
-    goToDatasetDetails(props) {
-        this.props.dispatch(push(`${location.pathname}/dataset/${props.id}`));
+        return values;
     }
 
     render() {
         return (
             <div className="grid grid--justify-center">
-                <div className="grid__col-4">
-                    <h1 className="text-center">Select a dataset</h1>
-                    <div className="margin-bottom--1">
-                        <Link to={`${location.pathname}/uploads`}>Upload a dataset</Link>
-                    </div>
-                    <SelectableBoxController
-                        heading="Instances"
-                        isUpdating={this.state.isFetchingDatasets}
-                        handleItemClick={this.goToDatasetMetadata}
-                        items={this.state.datasets}
-                    />
-                    <div className="margin-top--1">
-                      <SelectableBoxController
-                          heading="Datasets"
-                          isUpdating={this.state.isFetchingDatasets}
-                          handleItemClick={this.goToDatasetDetails}
-                          items={this.state.allDatasets}
-                      />
+                <div className="grid__col-7">
+                    <ul className="list list--neutral">
+                        <li className="list__item grid grid--justify-space-between">
+                            <h1>Select a dataset</h1>
+                            <div className="margin-top--3">
+                                <a className="btn btn--primary" href={`${location.pathname}/uploads`}>Upload a dataset</a>
```

Comment:
This should use the <Link> component rather than an <a> element (eg https://github.com/ONSdigital/florence/blob/099a9bd7f9e5e6c377484ae822a8c0049d549085/src/app/views/datasets/DatasetsController.jsx#L153). Using the <a> causing the page to do a full page reload, rather than rendering the next route with React without a hard load.